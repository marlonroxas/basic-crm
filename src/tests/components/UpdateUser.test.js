import React from 'react'
import { shallow } from 'enzyme'

import UpdateUser from '../../components/UpdateUser/index'

test('Should render UpdateUser correctly', () => {
  const wrapper = shallow(<UpdateUser />)
  expect(wrapper).toMatchSnapshot()
})