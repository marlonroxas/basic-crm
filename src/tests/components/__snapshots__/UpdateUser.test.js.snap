// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render UpdateUser correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <label>
      First Name
    </label>
    <input
      name="first_name"
      onChange={[Function]}
      required={true}
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Last Name
    </label>
    <input
      name="last_name"
      onChange={[Function]}
      required={true}
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Date of Birth
    </label>
    <input
      name="birthdate"
      onChange={[Function]}
      required={true}
      type="date"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Email
    </label>
    <input
      name="email"
      onChange={[Function]}
      required={true}
      type="email"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Phone
    </label>
    <input
      name="phone"
      onChange={[Function]}
      required={true}
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Address
    </label>
    <input
      name="address"
      onChange={[Function]}
      required={true}
      type="text"
    />
  </div>
  <div
    className="form-group form-group--actions"
  >
    <button
      className="primary-btn"
    >
      Update
    </button>
    <button
      className="cancel-btn"
      onClick={[Function]}
    >
      Cancel
    </button>
  </div>
</form>
`;
